ebml_matroska.xml

```diff
@@ -885,7 +885,9 @@ redundant framing information while preserving versioning and semantics between
 
 Value represents a clockwise rotation, in degrees, around the up vector. This rotation must be applied
 before any `ProjectionPosePitch` or `ProjectionPoseRoll` rotations.
-The value of this field should be in the -180 to 180 degree range.</documentation>
+The value of this field should be in the -180 to 180 degree range.
+
+Setting `ProjectionPoseYaw` to 180.0 degrees flips the image horizontally.</documentation>
```

Comment:
In general I'm fine with the change. But what's the difference between -180 and 180? Why recommend 180 but not -180?

In addition: please use `180` instead of `180.0`. The prior sentence states that `180` is the limit anyway; therefore any decimal part than 0 would be outside the range anyway.

Reply:
Fixed by allowing negative values and removing the decimals.